addons/TBMod_arsenal/functions/fn_loadDefault.sqf

```diff
@@ -112,7 +113,7 @@ if (_arsenalType == "CUSTOM") exitWith
 {
     if (isNil "TB_fnc_customLayout") then
     {
-        TB_fnc_customLayout = {
+        TB_fnc_customLayout = {//Not changed for backward comp.
```

Comment:
Was soll ich dafür tun?

Reply:
denke mal lassen... ist in allen init.sqfs drin